massa-network-worker/src/peer_info_database.rs

```diff
@@ -209,10 +212,11 @@ impl PeerInfoDatabase {
         let peers_file_dump_interval = cfg.peers_file_dump_interval;
         let (saver_watch_tx, mut saver_watch_rx) = watch::channel(peers.clone());
         let mut need_dump = false;
-        let saver_join_handle = tokio::spawn(async move {
+        let saver_join_handle = runtime.spawn(async move {
```

Comment:
[`tokio::spawn`](https://docs.rs/tokio/latest/tokio/fn.spawn.html) needs to be called from an async context, so that it just uses the runtime in which the current task is running. So since we're removing async from the "top-level" of network, there will not necessarily be a async context to call `spawn` from. 

So it's better for network to own it's own runtime, and use it directly to spawn tasks.

Reply:
Ok that's clear. Thank you